### PR TITLE
Fixes & websites adding

### LINKED ISSUE #4107: [Site Request] Eternal Scans
**Name of the website**
Eternal Scans

**Website urls (examples below)**
- Site: https://eternalscans.com/
- Manga List: https://eternalscans.com/series/
- Manga example: https://eternalscans.com/series/the-dragons-brave-love/
- Chapter online viewer example: https://eternalscans.com/series/the-dragons-brave-love/chapter-5/

**Languages**
English

**Website relationship*
If applicable describe the relation with any other website (eg : "alternative domain, copy of ...").

**Additional details**
Add any other context details that you may have found (like template to reuse)

### LINKED ISSUE #4108: [Site Request] mangaboss
**Name of the website**
mangaboss

**Website urls (examples below)**
- Site: https://mangaboss.org/
- Manga List: https://mangaboss.org/manga/
- Manga example: https://mangaboss.org/manga/rebirth-of-god-level-prodigal-son/
- Chapter online viewer example: https://mangaboss.org/manga/rebirth-of-god-level-prodigal-son/chapter-12/

**Languages**
Ebglish

**Website relationship*
If applicable describe the relation with any other website (eg : "alternative domain, copy of ...").

**Additional details**
Add any other context details that you may have found (like template to reuse)

### LINKED ISSUE #4162: [Site Request] Minda Fansub
**Name of the website**
Minda Fansub

**Website urls**
- Site: https://mindafansub.me/
- Manga List: https://mindafansub.me/manga/list-mode/
- Manga example: https://mindafansub.me/manga/demons-sweetheart/
- Chapter online viewer example: https://mindafansub.me/demons-sweetheart-1-bolum/

**Languages**
Turkish

**Website relationship**
No

**Additional details**
No

### LINKED ISSUE #4168: [Site Request] muctau
**Name of the website**
muctau

**Website urls (examples below)**
- Site: https://muctau.com/
- Manga List: https://muctau.com/manga/
- Manga example: https://muctau.com/manga/a-love-you-cant-escape/
- Chapter online viewer example: https://muctau.com/manga/a-love-you-cant-escape/chapter-12/

**Languages**
English

**Website relationship*
If applicable describe the relation with any other website (eg : "alternative domain, copy of ...").

**Additional details**
Add any other context details that you may have found (like template to reuse)

### LINKED ISSUE #4178: [Site Request] unemployed scans
**Name of the website**
unemployed scans

**Website urls (examples below)**
- Site: https://unemployedscans.com/
- Manga List: https://unemployedscans.com/manga/
- Manga example: https://unemployedscans.com/manga/brother-is-not-a-villain/
- Chapter online viewer example: https://unemployedscans.com/manga/brother-is-not-a-villain/chapter-16/

**Languages**
English

**Website relationship*
If applicable describe the relation with any other website (eg : "alternative domain, copy of ...").

**Additional details**
Add any other context details that you may have found (like template to reuse)

### LINKED ISSUE #4225: [Site Request] Azora World
**Name of the website**
Azora World

**Website urls**
- Site: https://azoraworld.com/
- Manga List: https://azoraworld.com/series/
- Manga example: https://azoraworld.com/series/slave-b/
- Chapter online viewer example: https://azoraworld.com/series/slave-b/44/

**Languages**
Arabic

**Additional details**
Provide: Manga - Manhua - Manhwa

### LINKED ISSUE #4228: [Jiangzaitoon] Connector not working
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
Website is changed

**To Reproduce**
1. https://jiangzaitoon.net/
2. https://jiangzaitoon.net/manga/cansiz-adam/
3. https://jiangzaitoon.net/manga/cansiz-adam/bolum-23-final/

**HakuNeko (please complete the following information):**
 - Version 6.1.7

### LINKED ISSUE #4229: [Opiatoon] Connector not working
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
Website is changed.

**To Reproduce**
1. https://opiatoon.net/ (old one https://opiatoon.com)
2. https://opiatoon.net/manga/beware-of-demons
3. https://opiatoon.net/manga/beware-of-demons/bolum-0/

**HakuNeko (please complete the following information):**
 - Version 6.1.7

### LINKED ISSUE #4235: [Site Request] Aqua Manga
**Name of the website**
Aqua Manga

**Website urls (examples below)**
- Site: https://aquamanga.com/
- Manga List: https://aquamanga.com/manga/
- Manga example: https://aquamanga.com/read/black-clover/
- Chapter online viewer example: https://aquamanga.com/read/black-clover/chapter-317/

**Languages**
English

**Website relationship*
independent

**Additional details**
Add any other context details that you may have found (like template to reuse)

### LINKED ISSUE #4257: [olympusscanlation.com] Connector not working
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes/N

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
Error 404
**To Reproduce**
1. olympusscanlation.com
2. Any
3. Any
4. Error 404 and CLIPBOARD: https://olympusscanlation.com/manga/el-hombre-fuerte-del-manicomio/ TypeError: Cannot read property 'textContent' of undefined
    at OlympusScanlation._getMangaFromURI (mjs/connectors/templates/WordPressMangastream.mjs:81)
    at async hakuneko:/cache/mjs/connectors/system/ClipboardConnector.mjs:64
    at async Promise.all (index 0)
**Screenshots**
If complex to describe, add screenshots to help explain your problem.

**HakuNeko (please complete the following information):**
 - Version [e.g. 6.17]

### PR DESCRIPTION
**Pull request by onelots**
New Pr for a bunch of fixes & add : 

=> Fixed Opiatoon (#4229)
=> Fixed JiangZaiToon (#4228)
=> Fixed Olympusscanlation (#4257)

=> Added Mangaboss (#4108)
=> Added EternalScans (#4107)
=> Added AquaManga (#4235)
=> Added AzoraWorld (#4225)
=> Added UnemployedScans (#4178)
=> Added Muctau (#4168) 
=> Added Mindafansub (#4162) 

Enjoy and thanks to the hakuneko's dev for their hard work !

### CODE REVIEW

File: src/web/mjs/connectors/azoraworld.mjs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class AzoraWorld extends WordPressMadara {
+
+    constructor() {
+        super();
+        super.id = 'azoraworld';
+        super.label = 'AzoraWorld';
+        this.tags = [ 'webtoon', 'turkish', 'manga' ];
```
Comment (27136773) [inline src/web/mjs/connectors/azoraworld.mjs]: Isn't it an `arabic` website?

**Review by 27136773 — APPROVED** [review]
Looks technically good

File: src/web/mjs/connectors/azoraworld.mjs (hunk 1 above)
Comment (onelots) [reply src/web/mjs/connectors/azoraworld.mjs]: Aha yeah u're right, I messed up X)
I'm gonna change it.

**Comment by onelots** [comment]
Azoraworld has been corrected, it was arabic as u said.

File: src/web/mjs/connectors/azoraworld.mjs (hunk 1 above)
Comment (onelots) [reply src/web/mjs/connectors/azoraworld.mjs]: I changed it, that's good.

**Pull request merged**